components/common/EmptyResult.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <section class="px-6 py-12 text-center">
+  <section class="px-6 py-12 mb-16 text-center">
```

Comment:
let's stay consistent and use padding

Reply:
done

Reply:
actually here it's changing for every component that uses `ItemsGrid.vue`, 

you should add any extra margin inside `GenerativeLayout`

![CleanShot 2024-04-11 at 11 26 55@2x](https://github.com/kodadot/nft-gallery/assets/44554284/d36cda00-a762-4ab8-b337-904e035bf714)

Reply:
ohhh thanks for the tip. it's better this way :)